src/pages/index.tsx

```diff
@@ -1,21 +1,52 @@
-import { Center, Heading, VStack } from '@chakra-ui/react';
+import {
+  Card,
+  Center,
+  Heading,
+  VStack,
+  CardBody,
+  Image,
+  Grid,
+} from '@chakra-ui/react';
 import { type NextPage } from 'next';
 
 import { api } from '../utils/api';
 
 const Home: NextPage = () => {
-  const hello = api.characterRouter.hello.useQuery({ name: 'Meraki' });
-  const count = api.characterRouter.count.useQuery();
+  const characters = api.characterRouter.getCharacters.useQuery()?.data;
 
   return (
-    <Center minH={'100vh'}>
+    <Center minH={'100vh'} backgroundColor="gray.900">
       <VStack spacing={5}>
-        <Heading as="h1" size="2xl">
-          {hello.data?.greeting}
-        </Heading>
-        <Heading as="h2" size="md">
-          There are {count?.data ?? 0} characters in the DB.
-        </Heading>
+        <Grid maxWidth="80%" templateColumns="repeat(3, 5fr)" gap={5}>
+          {characters &&
+            characters.map((character) => {
+              return (
+                <a href={`/character/${character.id}`} key={character.name}>
+                  <Card width="100%" borderRadius={0}>
```

Comment:
Chakra UI has abbreviations for a lot of these attributes, like `w="100%"`, `maxW="80%"`, etc. [Here's](https://chakra-ui.com/docs/styled-system/style-props) a list!